### PR TITLE
Centralised packages

### PR DESCRIPTION
**Pull request by EwanNoble**

### CODE REVIEW

**Review by dashton82 — DISMISSED** [review]

File: src/ApimDeveloper/SFA.DAS.ApimDeveloper.UnitTests/SFA.DAS.ApimDeveloper.UnitTests.csproj [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
         </PackageReference>
-        <PackageReference Include="FluentAssertions" Version="5.10.3" />
-        <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.0.0" />
-        <PackageReference Include="NUnit3TestAdapter" Version="3.13.0" />
-        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
-        <PackageReference Include="SFA.DAS.Testing.AutoFixture" Version="3.0.106" />
-        <PackageReference Include="System.ComponentModel.TypeConverter" Version="4.3.0" />
+        <PackageReference Include="FluentAssertions" />
+        <PackageReference Include="Microsoft.Extensions.Configuration.Json" />
+        <PackageReference Include="Moq" />
```
Comment (NickGraham101) [inline src/ApimDeveloper/SFA.DAS.ApimDeveloper.UnitTests/SFA.DAS.ApimDeveloper.UnitTests.csproj:15]: Why has a new package been introduced?

File: src/EmployerIncentives/SFA.DAS.EmployerIncentives.UnitTests/SFA.DAS.EmployerIncentives.UnitTests.csproj [hunk 2]
```diff
@@ -7,17 +7,18 @@
...
           <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
-        <PackageReference Include="FluentAssertions" Version="5.10.3" />
-        <PackageReference Include="nunit" Version="3.12.0" />
-        <PackageReference Include="NUnit3TestAdapter" Version="3.15.1" />
-        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.4.0" />
-        <PackageReference Include="SFA.DAS.Testing.AutoFixture" Version="3.0.106" />
-        <PackageReference Include="System.ComponentModel.TypeConverter" Version="4.3.0" />
+        <PackageReference Include="FluentAssertions" />
+        <PackageReference Include="Moq" />
```
Comment (NickGraham101) [inline src/EmployerIncentives/SFA.DAS.EmployerIncentives.UnitTests/SFA.DAS.EmployerIncentives.UnitTests.csproj:16]: Introduced a new package here as well

File: src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement.UnitTests/SFA.DAS.RoatpCourseManagement.UnitTests.csproj [hunk 3]
```diff
@@ -7,19 +7,20 @@
...
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="FluentAssertions" Version="6.6.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
-    <PackageReference Include="NUnit" Version="3.13.2" />
-    <PackageReference Include="NUnit3TestAdapter" Version="4.0.0" />
-    <PackageReference Include="coverlet.collector" Version="3.1.2">
+    <PackageReference Include="FluentAssertions" VersionOverride="6.6.0" /> <!-- TODO: Align with central package versions -->
+    <PackageReference Include="Microsoft.NET.Test.Sdk" />
+    <PackageReference Include="Moq" />
```
Comment (NickGraham101) [inline src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement.UnitTests/SFA.DAS.RoatpCourseManagement.UnitTests.csproj:16]: Introduced new package

File: src/VacanciesManage/SFA.DAS.VacanciesManage.UnitTests/SFA.DAS.VacanciesManage.UnitTests.csproj [hunk 4]
```diff
@@ -6,16 +6,17 @@
...
     </PackageReference>
-    <PackageReference Include="FluentAssertions" Version="5.10.3" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.0.0" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.13.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
-    <PackageReference Include="SFA.DAS.Testing.AutoFixture" Version="3.0.106" />
-    <PackageReference Include="System.ComponentModel.TypeConverter" Version="4.3.0" />
+    <PackageReference Include="FluentAssertions" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" />
+    <PackageReference Include="Moq" />
```
Comment (NickGraham101) [inline src/VacanciesManage/SFA.DAS.VacanciesManage.UnitTests/SFA.DAS.VacanciesManage.UnitTests.csproj:15]: Introduced new package

**Review by dashton82 — APPROVED** [review]

**Review by NickGraham101 — APPROVED** [review]

**Pull request merged**